templates/webpack/webpack.custom.ts

```diff
@@ -296,8 +296,8 @@ export default (config: Configuration, angularJsonConfig: CustomWebpackBrowserSc
     });
   });
 
-  if (theme !== 'default') {
-    const defaultThemePath = join('src', 'styles', 'themes', 'default');
+  if (theme !== 'b2b') {
+    const defaultThemePath = join('src', 'styles', 'themes', 'b2b');
```

Comment:
this `if` must now be removed and the `'b2b'` must become `'placeholder'`, otherwise the component style imports won't work.

You can test that by adding a component style override with `@import 'variables';`.
I think this test can also be added to the `test-schematics-customization.sh`

Reply:
see 61b006e